style.css

```diff
@@ -0,0 +1,107 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+/**
+*! Color variables
+**/
+
+/**
+*! Removing default browser style
+**/
+* {
+  box-sizing: border-box;
+}
+
+* body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+* a {
+  text-decoration: none;
+}
+
+* li {
+  list-style: none;
+}
+
+/**
+*! Header styles inputs here
+**/
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 1.25rem;
+  margin-left: 1rem;
+  color: #6070ff;
+}
+
+[class^=bar] {
+  display: block;
+  background-color: #6070ff;
+  width: 1.875rem;
+  height: 0.1875rem;
+  margin: 0.3125rem;
+  margin-left: 14rem;
+}
+
+/**
+*! Main section styles inputs here
+**/
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./images/header-shapes mobile.svg");
+  background-repeat: no-repeat;
+  background-position: top left;
+  background-color: white;
+  width: 23.4375rem;
```

Comment:
- [ ] I suggest avoiding setting a width even if the unit used here is a relative unit it will not successfully make the design responsive.
It will be great if the browser decided the width of this element based on the screen size.